Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -1094,6 +1097,8 @@
     <RegEx find="stomačn" replaceWith="trbušn" />
     <RegEx find="Stomačn" replaceWith="Trbušn" />
     <RegEx find="\b([sS])trj?el([aeiouc])" replaceWith="$1trijel$2" />
+    <RegEx find="sugeriše([mšt])" replaceWith="predlaže$2" />
+    <RegEx find="Sugeriše([mšt])" replaceWith="Predlaže$2" />
```

Comment:
@diomed Replacement pattern check added in [1.1.3](https://github.com/xylographe/SE-WordListValidator/releases/tag/1.1.3)